<launch>

<arg name="fixed_frame" default="world"/>
<arg name="odom_frame" default="odom"/>

<include file="$(find ekf_ukf)/launch/noisy_odom.launch"></include>
<node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher"/>

<node pkg="tf" type="static_transform_publisher" name="odom_tf" args="0 0 0 0 0 0 /$(arg fixed_frame) /$(arg odom_frame) 10"/>

</launch>
